ray-operator/apis/ray/v1/rayjob_types.go

```diff
@@ -64,6 +71,10 @@ type RayJobSpec struct {
 	RayClusterSpec *RayClusterSpec `json:"rayClusterSpec,omitempty"`
 	// clusterSelector is used to select running rayclusters by labels
 	ClusterSelector map[string]string `json:"clusterSelector,omitempty"`
+	// If LightWeightSubmissionMode is true, KubeRay operator sends a request to the RayCluster to create a
```

Comment:
Doc string here needs update

Reply:
Good catch. I forgot to update the comment.